config/database.rb

```diff
@@ -1,13 +1,35 @@
 configure do
-  # Log queries to STDOUT in development
-  if Sinatra::Application.development?
+
+  case Sinatra::Application.environment
```

Comment:
I think the skeleton was setup be show the programmatic ways initially, takes some of the magic away so they understand how it hooks up?